client/landing/stepper/hooks/use-site-migraiton-key.ts

```diff
@@ -0,0 +1,22 @@
+import { useQuery } from '@tanstack/react-query';
```

Comment:
Nit: the filename has a typo. `-migraiton-` => `-migration-`

Reply:
Fixed https://github.com/Automattic/wp-calypso/pull/88952/commits/694f4fccc9098322d923d169e7b9f886fe9d15f3